File: Problem/Failed/Programmers/Hash/Level_1/PG_42576/Solution.java
import java.io.BufferedReader;
import java.io.FileInputStream;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.Arrays;
import java.util.LinkedList;
import java.util.StringTokenizer;

class Solution {
  // -- Local Input ====================
  public static void main(String[] args) throws IOException, NumberFormatException {
    System.setIn(new FileInputStream("question/input.txt"));
    Solution solInstance = new Solution();

    int[][] input;
    
    {
      LinkedList<LinkedList<Integer>> rawInput = new LinkedList<LinkedList<Integer>>();
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      StringTokenizer st = new StringTokenizer(br.readLine(), "[, ]");

      while(st.hasMoreTokens()) {
        rawInput.add(
          new LinkedList<Integer>(Arrays.asList(
            Integer.parseInt(st.nextToken()), 
            Integer.parseInt(st.nextToken())
          ))
        );
      }
      br.close();
      input = rawInput.stream().map(e -> e.stream().mapToInt(i -> i).toArray()).toArray(int[][]::new);
    }

    System.out.println("[answer]: " + solInstance.solution(input));
    return;
  }

  // -- Solution =======================
  /**
   * java Solution.java
   * 2차원 배열 데이터 전용
   */
  public int solution(int[][] datas) {
    int answer = 0;

    for(int[] data : datas) System.out.println(new StringBuffer()
      .append('[')
      .append(data[0])
      .append(", ")
      .append(data[1])
      .append(']')
    );

    return answer;
  }
}

File: Problem/Failed/Programmers/Hash/Level_1/PG_42576/Solution.js
// -- Reader Object ==================
const reader = {
  hasNextLine: () => !!this.fullLine[this.lineIndex],
  readLine: (() => {
    this.lineIndex = 0;
    this.fullLine = require('fs')
      .readFileSync('question/input.txt')
      .toString()
      .split(/\r\n|\n|\r/gm);
    return () => this.fullLine[this.lineIndex++];
  })(),
};

// -- Local Input ====================
let input = [];

let iter = reader.readLine().matchAll(/[0-9]+/g);

for (let curr = iter.next(); !curr.done; curr = iter.next()) {
  input.push([curr.value[0], iter.next().value[0]]);
}

console.log('[answer]: ', solution(input));

// -- Solution =======================
/**
 * node Solution.js
 */
function solution(datas) {
  let answer = 0;

  datas.forEach((e) => {
    console.log(`[${e[0]}, ${e[1]}]`);
  });

  return answer;
}
